src/components/user/UserCard.jsx

```diff
@@ -1,7 +1,33 @@
-import React from "react";
+import React, { useEffect } from "react";
 import Button from "../Button.jsx";
+import { deleteParticipant } from "../../comutils";
+import { toast } from "react-toastify";
 
-const UserCard = ({ user, userUuid, currentUserUuid }) => {
+const UserCard = ({
+    user,
+    userUuid,
+    currentUserUuid,
+    planId,
+    adminId,
+    participants,
+    setParticipants,
+}) => {
+    const leavePlan = () => {
+        if (userUuid === currentUserUuid || currentUserUuid === adminId) {
+            deleteParticipant(planId, userUuid).then((r) => {
+                toast.success("Participant deleted");
+                // Ejemplo: Eliminar la fila con id igual a 2
```

Comment:
Please remove this comment or translate if you think it's relevant.

Reply:
well, this is the language of Cristo nuestro señor, but will remove it for you, no worries